### ISSUE TITLE
[Bug] v.in.ogr and v.import windows crash in Linux (GRASS 7.8.5)

### ISSUE THREAD

**Issue by cmbarton** [OSGeo/grass#1417]
**Describe the bug**
Selecting v.import or v.in.ogr from the wxGUI menu, OR typing v.in.ogr and pressing return in the terminal produces the following error.

```
v.in.ogr
Traceback (most recent call last):
  File "/usr/lib/grass78/gui/wxpython/gui_core/forms.py", line 3005, in OnInit
    self.mf = TaskFrame(
  File "/usr/lib/grass78/gui/wxpython/gui_core/forms.py", line 546, in __init__
    self.notebookpanel = CmdPanel(
  File "/usr/lib/grass78/gui/wxpython/gui_core/forms.py", line 2035, in __init__
    win = gselect.GdalSelect(parent=parent, panel=which_panel,
  File "/usr/lib/grass78/gui/wxpython/gui_core/gselect.py", line 1520, in __init__
    fileFormats = GetFormats(writableOnly=dest)[fType]['file']
  File "/usr/lib/grass78/gui/wxpython/core/utils.py", line 713, in GetFormats
    gdalAll, gdalWritable = _getGDALFormats()
  File "/usr/lib/grass78/gui/wxpython/core/utils.py", line 645, in _getGDALFormats
    return _parseFormats(ret), _parseFormats(ret, writableOnly=True)
  File "/usr/lib/grass78/gui/wxpython/core/utils.py", line 675, in _parseFormats
    key, name = map(lambda x: x.strip(), line.strip().rsplit(':', -1))
ValueError: too many values to unpack (expected 2)
OnInit returned false, exiting...
```

Other GUI windows work fine.

**System description (please complete the following information):**

- Operating System: 
 Ubuntu 20.04 focal
 Kernel: x86_64 Linux 5.4.0-66-generic
 Uptime: 2m
 Packages: 3587
 Shell: bash 5.0.17
 Resolution: 2560x1080
 DE: KDE 5.68.0 / Plasma 5.18.5
 WM: KWin
 GTK Theme: Breeze [GTK2/3]
 Icon Theme: breeze
 Disk: 1.2T / 1.9T (67%)
 CPU: Intel Core i5-4460 @ 4x 3.4GHz [39.0°C]
 GPU: GeForce GTX 1050 Ti
 RAM: 2429MiB / 15954MiB

- GRASS GIS version: 7.8.5

**Comment by petrasovaa** [comment]
Already addressed in #1226 and backported.

**Comment by cmbarton** [comment]
> Already addressed in #1226 and backported.

I was hoping that was the case. But I did not find anything searching on v.in.ogr
